javascripts/discourse/templates/components/top-contributors.hbs

```diff
@@ -10,7 +10,11 @@
         class="top-contributors--user-badge"
       >
         {{avatar item.user imageSize="small"}}
-        {{html-safe item.user.username}}
+        {{#if (and item.user.name siteSettings.enable_names)}}
+          {{html-safe item.user.name}}
```

Comment:
I know it was already there, but I think we shouldn't use `html-safe` here.

Reply:
Does this apply for just `name` or should I get rid of it for `username` too?

Reply:
I'd get rid of it for both.

The `html-safe` name isn't very clear about it, but what it does effectively is ignore any HTML escaping, assuming whatever content we have there is a safe HTML that has been sanitized/validated elsewhere.

When we DO need it, we need to properly validate the input against malicious scripts. In this particular case, I don't think it would be a big risk, but if we don't even need to render it as HTML, IMO there's no point in keeping them behind a `html-safe`.